src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -372,14 +389,20 @@ public void elementShouldNotBeClickable(String locator, String...params) {
 	        "In contrast to `Element Should Contain`, this keyword does not try a substring match but an exact match on the element identified by locator.\r\n" + 
 	        "\r\n" + 
 	        "Key attributes for arbitrary elements are id and name. See `Introduction` for details about locators.")
-	@ArgumentNames({ "locator", "text", "message=NONE" })
+	@ArgumentNames({ "locator", "text", "message=NONE", "ignore_case=False" })
 	public void elementTextShouldBe(String locator, String text, String...params) {
         String message = robot.getParamsValue(params, 0, "");
+		Boolean ignoreCase = robot.getParamsValue(params, 1, Boolean.FALSE);
 		List<WebElement> elements = elementFind(locator, true, true);
 		String actual = elements.get(0).getText();
+		String textToCompare = text;
+		if (ignoreCase) {
+			actual = actual.toLowerCase();
+			textToCompare = textToCompare.toLowerCase();
```

Comment:
String class and StringUtils have methods for case insensitive comparison, perhaps, we could use them. WDYT?

* https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#equalsIgnoreCase(java.lang.String)
* https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#equalsIgnoreCase-java.lang.CharSequence-java.lang.CharSequence-

Reply:
I kep that toLower from the original code of the class. So I'm fine with any way you want to change that. The main thing that needed adding was the option to chooise between testing with and without case sensitivity.

Reply:
I'm changing it to:

```
String actual = elements.get(0).getText();
if (!(ignoreCase && text.equalsIgnoreCase(actual)) || !(!ignoreCase && text.equals(actual))) {
}
```